example/lib/src/file_picker_demo.dart

```diff
@@ -1,7 +1,8 @@
 import 'package:file_picker/file_picker.dart';
-import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
+import 'dart:convert';
+import 'dart:io' as io;
```

Comment:
Don't import `dart:io`. This would break the example when run on the web.

Instead, use https://pub.dev/packages/file in the example app, and guard it with `kIsWeb`.

Reply:
I have replaced it with `import 'package:file/local.dart';` and execute the code only when we're not on web. Is that what you meant?

```dart
...

          String name = 'File path:';
          if (!kIsWeb) {
            final fs = LocalFileSystem();
            name = fs.isFileSync(pickedFilesAndDirectories![index])
                ? 'File path:'
                : 'Directory path:';
          }

...
```

Reply:
Exactly!